hnn_core/parallel_backends.py

```diff
@@ -125,8 +125,10 @@ def run_subprocess(command, obj, timeout, proc_queue=None, *args, **kwargs):
     threads_started = False
 
     try:
-        proc = Popen(command, stdin=PIPE, stdout=PIPE, stderr=PIPE, *args,
-                     **kwargs)
+        kwargs.setdefault('stdin', PIPE)
+        kwargs.setdefault('stdout', PIPE)
+        kwargs.setdefault('stderr', PIPE)
+        proc = Popen(command, *args, **kwargs)
```

Comment:
How was this fixing a flake8 error? L128 was well under 79 characters....

Reply:
It wasn't a line length issue. The error was [B026](https://docs.astral.sh/ruff/rules/star-arg-unpacking-after-keyword-arg/), which warns against using star-argument unpacking after keyword arguments.

Reply:
This is less readable IMO ...

Reply:
I agree that this is less readable, but I also see the issue with unpacking star-arguments after keyword arguments.

What if we were to do a hard update to `kwargs` with these three key/val pairs instead of setting defaults? It's minor, but also slightly more explicit.